build.sbt

```diff
@@ -45,8 +45,9 @@ libraryDependencies ++= Seq(
   "com.amazonaws" % "aws-java-sdk-stepfunctions" % "1.11.128",
   "org.typelevel" %% "cats" % "0.9.0",
   "play-circe" %% "play-circe" % "2.6-0.8.0",
-  "com.gu" %% "support-models" % "0.14",
+  "com.gu" %% "support-models" % "0.15-SNAPSHOT",
```

Comment:
This should be 0.15. We are waiting for the release of `support-models` to update it.